c_glib/test/test-chunked-array-datum.rb

```diff
@@ -49,7 +49,10 @@ def test_false
   end
 
   def test_to_string
-    assert_equal("ChunkedArray([\n" + "  [\n" + "    true,\n" + "    false\n" + "  ]\n" + "])", @datum.to_s)
+    assert_equal("ChunkedArray([\n" +
+                 "  [\n" + "    true,\n" +
+                 "    false\n" + "  ]\n" + "])",
+                 @datum.to_s)
```

Comment:
Let's use here-document.

```suggestion
    assert_equal(<<-STRING.chomp, @datum.to_s)
ChunkedArray([
  [
    true,
    false
  ]
]
                 STRING
```

Reply:
Thanks! Fixed.